src/app/common/utils/resolvers/pages-wallet-details.resolver.spec.ts

```diff
@@ -53,4 +56,18 @@ describe('PagesWalletDetailsResolver', () => {
       });
     });
   });
+
+  describe('navigate to 404 error page', () => {
+    describe('success', () => {
+      it('should navigate to /** - error 404 page after passing undefined wallet', () => {
+        resolver.navigateErrorPageIfWalletNotFound(undefined);
```

Comment:
You should not test `navigateErrorPageIfWalletNotFound` as this method should be private. You should test `resolve` method.